src/styles/card-style.scss

```diff
@@ -0,0 +1,90 @@
+.container__card {
```

Comment:
why your file named "card-style" if you styling here container class?
card supposed to be a bem block, like in the previous task